pkgs/top-level/python-packages.nix

```
@@ -5719,6 +5719,10 @@ in {
 
   nltk = callPackage ../development/python-modules/nltk { };
 
+  nnpdf = toPythonModule (pkgs.nnpdf.override {
+    python3 = python;
```

If I remember correctly only
```suggestion
    python3 = self.python;
```
works.

---

I doubt it makes any difference
```
# ag python3 pkgs/top-level/python-packages.nix 
42:    inherit namePrefix;     # We want Python libraries to be named like e.g. "python3.6-${name}"
552:  antlr4-python3-runtime = callPackage ../development/python-modules/antlr4-python3-runtime {
584:  appleseed = toPythonModule (pkgs.appleseed.override { python3 = self.python; });
833:  avro-python3 = callPackage ../development/python-modules/avro-python3 { };
3317:    python3 = python;
4626:    python3 = python;
4767:    python3 = python;
4790:          outputsToInstall = [ "py" ]; # The package always builds python3 bindings
4794:    (p: p.override { python3 = python; })
4851:        python3 = python;
4929:    python3 = python;
5205:  mecab-python3 = callPackage ../development/python-modules/mecab-python3 { };
5226:    python3Packages = self;
5245:  meson = toPythonModule ((pkgs.meson.override { python3 = python; }).overrideAttrs
5293:    python3 = python;
5335:    python3 = python;
5669:    python3 = python;
5674:    python3Packages = self;
5723:    python3 = python;
7884:    python3 = self.python;
8277:  python3-application = callPackage ../development/python-modules/python3-application { };
8279:  python3-eventlib = callPackage ../development/python-modules/python3-eventlib { };
8281:  python3-gnutls = callPackage ../development/python-modules/python3-gnutls { };
8283:  python3-openid = callPackage ../development/python-modules/python3-openid { };
8287:  python3-saml = callPackage ../development/python-modules/python3-saml { };
8375:    meson = pkgs.meson.override { python3 = self.python; };
8944:    python3Packages = self;
9096:    python3 = python;
9356:    python3 = self.python;
11312:    python3 = python;
11318:    python3 = python;
```